.github/workflows/merge-master.yml

```diff
@@ -0,0 +1,108 @@
+name: Tag Merge to Master
+
+on:
+  push:
+    branches:
+    - master
+
+env:
+  CGO_ENABLED: "0"
+
+jobs:
+
+  tag:
+    name: Bump semantic version
+    runs-on: [ubuntu-latest]
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - uses: anothrNick/github-tag-action@1.17.2
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        DEFAULT_BUMP: patch
+        WITH_V: true
+
+  build:
```

Comment:
Oops I need to add the build matrix for Windows and MacOS if you also want to support those OSs.